packages/manager/apps/web/client/app/hosting/multisite/delete/hosting-multisite-delete.controller.js

```diff
@@ -98,6 +98,36 @@ angular.module('App').controller(
         '#/hosting/:serviceName/change-offer',
         { serviceName: this.$stateParams.productId },
       );
+
+      this.listDomainWithSamePath();
+    }
+
+    listDomainWithSamePath() {
+      this.HostingDomain.getAttachedDomains(this.$stateParams.productId, {
+        params: {
+          path: this.selected.domain.path,
+        },
+      })
+        .then((data) => {
+          this.domainsWithSamePath = data;
+          this.checkDomainWithSamePath();
+        })
+        .finally(() => {
+          this.isLoading = false;
+        });
+    }
+
+    checkDomainWithSamePath() {
+      this.hasDomainWithSamePath =
+        this.domainsWithSamePath.filter((domain) => {
+          if (this.selected.wwwNeeded) {
+            return (
+              domain !== `www.${this.selected.domain.name}` &&
+              domain !== this.selected.domain.name
+            );
+          }
+          return domain !== this.selected.domain.name;
+        })?.length > 0;
```

Comment:
```suggestion
        this.domainsWithSamePath.some((domain) => {
          if (this.selected.wwwNeeded) {
            return (
              domain !== `www.${this.selected.domain.name}` &&
              domain !== this.selected.domain.name
            );
          }
          return domain !== this.selected.domain.name;
        });
```
You could use some instead (that way you won't iterate over all the array if one element is matching the "filter")

Reply:
It's applied.